### PR TITLE
LPS-101678 Remove targetProperty and action props

### PR DESCRIPTION
**Pull request by brunofarache**

### CODE REVIEW

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/app/EditApp.es.js [hunk 1]
```diff
@@ -64,159 +42,81 @@ export default ({
...
-					<div className="card-header align-items-center d-flex justify-content-between bg-transparent">
-						<UpperToolbarInput
-							onInput={onAppNameChange}
-							placeholder={Liferay.Language.get('untitled-app')}
-							value={appName}
-						/>
-					</div>
-
-					<h4 className="card-divider mb-4"></h4>
+			<AppDeploymentProvider>
```
Comment (brunofarache) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/app/EditApp.es.js]: @Masterfoni better to rename to EditAppContextProvider as it's used in EditApp.

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/app/DeployApp.es.js [hunk 2]
```diff
@@ -12,32 +12,22 @@
...
-		});
-
-		setState(prevState => ({
-			...prevState,
-			[type]: !prevState[type]
-		}));
-	};
-
-	const {productMenu: isProductMenu} = state;
+	const productMenu = appDeployments.find(
```
Comment (brunofarache) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/app/DeployApp.es.js]: @Masterfoni replace with some()

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/app/EditApp.es.js [hunk 3]
```diff
@@ -13,44 +13,22 @@
...
 import EditAppBody from './EditAppBody.es';
 import EditAppFooter from './EditAppFooter.es';
 import MultiStepNav from './MultiStepNav.es';
-import Button from '../../components/button/Button.es';
 import ControlMenu from '../../components/control-menu/ControlMenu.es';
-import {UpperToolbarInput} from '../../components/upper-toolbar/UpperToolbar.es';
-import {addItem, updateItem} from '../../utils/client.es';
+import {AppDeploymentProvider} from './AppDeploymentContext.es';
+import EditAppHeader from './EditAppHeader.es';
+import DeployApp from './DeployApp.es';
```
Comment (brunofarache) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/app/EditApp.es.js]: @Masterfoni sort.

**Comment by patriciajeanperez** [comment]
Reviewed. No unique failures.

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/dc470dce8f...78b5898ccd

**Pull request closed without merging**